plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/ElementProblem.java

```diff
@@ -38,7 +38,7 @@
 
   /**
    * @param length the length of the marker underline. Length == 0 results in a
-   *        marker in the vertical ruler and no underline
+   *        marker in the vertical ruler and no underline.
```

Comment:
We don't place a period for `@param` and such.

Reply:
We do when it's at the end of a new sentence: Per https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format "The description begins with a lowercase letter if it is a phrase (contains no verb), or an uppercase letter if it is a sentence. End the phrase with a period only if another phrase or sentence follows it."

Yes, this is ultra-nitpicky. :-)